addons/pkg/constants/constants.go

```diff
@@ -97,4 +97,29 @@ const (
 
 	// CRDWaitPollTimeout is poll timeout for checking server resources
 	CRDWaitPollTimeout = time.Minute * 10
+
+	// DataValueFileName is the name of data value file
+	TKGDataValueFileName = "values.yaml"
+
+	// AntreaAddonName is the name of Antrea Addon Controller
+	AntreaAddonName = "antrea"
+)
+
+// infrastructure provider name constants
```

Comment:
nitpick: consider using constants already defined in cli code.

Reply:
Fixed